src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const tableHead = document.querySelector('thead');
+const cloneTableHead = tableHead.cloneNode(true);
+const tableFoot = document.querySelector('tfoot');
+const cloneTableFoot = tableFoot.cloneNode(true);
+const headPosition = cloneTableHead.querySelector('tr :nth-child(2)');
+const footPosition = cloneTableFoot.querySelector('tr :nth-child(2)');
+const headSalary = tableHead.querySelector('tr :nth-child(5)');
+const footSalary = tableFoot.querySelector('tr :nth-child(5)');
+const tableBody = document.querySelectorAll('tbody tr');
+
+headSalary.before(headPosition);
+footSalary.before(footPosition);
+
+tableBody.forEach((tr, index) => {
+  const secondChild = tr.querySelector(':nth-child(2)');
+  const fifthChild = tr.querySelector(':nth-child(5)');
+
+  fifthChild.before(secondChild.cloneNode(true));
+});
```

Comment:
You are cloning and inserting the second child of each row in the table body. However, you don't remove the original second child. This will result in duplicate cells in your table rows. You should remove the original second child after the clone has been inserted.